tests/test_serializers/test_json.py

```diff
@@ -298,8 +303,89 @@ def test_decode_path_error():
             JsonDeserializer.decode_path(value)
 
 
+class TestUnitEncoding:
+    """Tests for custom encoding of Unit types from pint."""
+
+    UNIT_DECODING_TEST_CASES = [
+        (
+            Unit("meter"),
+            CustomEncodedUnitTypes.unit.json_encoding("meter"),
+        ),
+    ]
+
+    UNIT_ENCODING_TEST_CASES = UNIT_DECODING_TEST_CASES
+
+    @staticmethod
+    def test_encode_unit_error():
+        """Test that the JsonSerializer.encode_unit raises an error if no encoding is defined."""
+        # Arrange
+        value = "foo"
+
+        # Act & Assert
+        with pytest.raises(NotImplementedError):
+            JsonSerializer.encode_unit(value)
+
+    @staticmethod
+    @pytest.mark.parametrize("value, expected", UNIT_ENCODING_TEST_CASES)
+    def test_encode_unit(value, expected):
+        """Test that the JsonSerializer.encode_unit encodes values as expected."""
+        # Arrange
```

Comment:
add this to # Act

Reply:
fixed [d2a3dfc](https://github.com/MORSECorp/snappiershot/pull/96/commits/d2a3dfc12a96bd7b9496c97b4b8a388d6995765e)